emloop/api.py

```diff
@@ -57,16 +58,23 @@ def create_output_dir(config: dict, output_root: str, default_model_name: str='U
             suffix += 1
 
     logging.info(f'Created output directory with name {output_dir}')
-    logging.info('\tOutput dir: %s', output_dir)
+    return output_dir
+
+
+def config_log_to_output_dir(config: dict, output_dir: str):
```

Comment:
- I truly can not comprehend this function name.
- is the `config` arg even used?
- what about
```python
os.makedirs(x, exist_ok=True)
```